install/helm/agones/templates/crds/_gameserverspecschema.yaml

```diff
@@ -177,18 +178,18 @@ properties:
         additionalProperties:
           type: object
           properties:
-            capacity: # max capacity of the array (can be less than or equal to value of maxItems)
+            capacity:
               type: integer
+              title: Max capacity of the array (can be less than or equal to value of maxItems)
               minimum: 0
               maximum: 1000 # must be equal to values.maxItems
             values:
+              title: set of all the items in the list
```

Comment:
@igooch PTAL at my changes here as well.

Also, not for this PR, in the original design the user couldn't set a default list value (like you can with counters) -- but it could be useful! Just wanted to highlight this to check that this was on purpose.

Reply:
Yes, they can set/populate all values to start (if the user wants).

Reply:
Neat! SGTM!